addons/sale_project/models/sale_order.py

```diff
@@ -17,8 +17,8 @@ class SaleOrder(models.Model):
     project_id = fields.Many2one(
         'project.project', 'Project', readonly=True, states={'draft': [('readonly', False)], 'sent': [('readonly', False)]},
         help='Select a non billable project on which tasks can be created.')
-    project_ids = fields.Many2many('project.project', compute="_compute_project_ids", string='Projects', copy=False, groups="project.group_project_user", help="Projects used in this sales order.")
-    project_count = fields.Integer(string='Number of Projects', compute='_compute_project_ids', groups='project.group_project_user')
+    project_ids = fields.Many2many('project.project', compute="_compute_project_ids", string='Projects', copy=False, groups="project.group_project_manager", help="Projects used in this sales order.")
+    project_count = fields.Integer(string='Number of Projects', compute='_compute_project_ids', groups='project.group_project_manager')
```

Comment:
I don't understand why it is the issue since a Project User can see the projects. 🤔

Reply:
As discussed with @xavierbol , `project_ids` is compute field so while setting value it will check the access rule.  so for the private project, it will raise the access error. Thanks